Common/Math/fast/fast_matrix.c

```diff
@@ -22,6 +22,42 @@ void fast_matrix_mul_4x4_sse(float *dest, const float *a, const float *b) {
 	}
 }
 
+#elif PPSSPP_ARCH(LOONGARCH64_LSX)
+
+typedef union
+{
+    int32_t i;
+    float f;
+} FloatInt;
+
+static __m128 __lsx_vreplfr2vr_s(float val)
+{
+    FloatInt tmpval = {.f = val};
```

Comment:
Nowadays it's more standard to use memcpy for this, compilers optimize it down properly. This usage of unions is UB. However, in practice it does work, so I'll allow it.